backend/app/search.py

```diff
@@ -27,6 +29,24 @@ def delete_apps(app_id_list):
         client.index("apps").delete_documents(app_id_list)
 
 
+def get_by_selected_categories(
+    selected_categories: List[schemas.Category], page: int, hits_per_page: int
+):
+    category_list = [
+        f"categories = {category.value}" for category in selected_categories
+    ]
+
+    return client.index("apps").search(
+        "",
+        {
+            "filter": [category_list],
+            "sort": ["installs_last_month:desc"],
+            "hitsPerPage": hits_per_page or 250,
```

Comment:
This might be problematic, if somebody ever tries to consume this without a limit, but all our pages have paging right now.

Reply:
Hrm, but we're doing paging client side right now, so this breaks it. Needs more work.